apps/server/src/modules/roster/service/feathers-roster.service.spec.ts

```diff
@@ -308,6 +308,8 @@ describe('FeathersRosterService', () => {
 				const courseC = courseEntityFactory.buildWithId();
 				const courses = [courseA, courseB, courseC];
 
+				// roomService.roomExists.mockResolvedValueOnce(false);
```

Comment:
That looks accidental.

Reply:
you are right, will remove that, was for testing purpose